app/Main.tsx

```
@@ -180,6 +180,9 @@ export function init() {
   const window = getWindow();
   const document = getDocument();
 
+  // Before error handling; being offline / not fetching announcements shouldn't display error message
+  getStore().dispatch(fetchAnnouncements());
```

I think it'd be safer to add a try/catch within fetchAnnouncements() that ensures all errors are suppressed. That way, we don't have to worry about the ordering of this code.

---

So, retried this - and turns out it's not actually an issue / it's a different part of the code throwing that error. See: https://github.com/ExpeditionRPG/expedition-app/issues/513